ansible/roles/vm_set/tasks/main.yml

```diff
@@ -182,26 +209,6 @@
         dest: "{{ root_path }}"
         mode: 0755
 
-    - name: Retrieve a list of the defined VMs
```

Comment:
why move up?

Reply:
The other code in the block is required for veos VM, but not for k8s Ubuntu VM. In k8s VM set up, I reference these registers/lists